eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver.php

```diff
@@ -265,13 +265,15 @@ private function warnAboutTooEarlyLoadedParams()
         }
 
         $logger = $this->container->get('logger');
-        foreach ($this->tooEarlyLoadedList as $param => $services) {
+        foreach ($this->tooEarlyLoadedList as $blame => $params) {
             $logger->warning(sprintf(
-                'ConfigResolver was used to load parameter "%s" before SiteAccess was loaded by services: %s. This can cause issues. '
-                . 'Try to use ConfigResolver lazily, '
-                . 'or try to mark the service as lazy.',
-                $param,
-                '"' . implode($services, '", "') . '"'
+                'ConfigResolver was used by "%s" before SiteAccess was initialized, loading parameter(s) '
+                . '%s. As this can cause very hard to debug issues, '
+                . 'try to use ConfigResolver lazily, '
+                // Symfony 3.4+: . (PHP_SAPI === 'cli' ? 'make the affected commands lazy, ' : '')
```

Comment:
I guess it's going to be uncommented on merge-up, right? 😉

Reply:
yes :)